cmd/logcli/main.go

```diff
@@ -76,7 +76,54 @@ data points between the start and end query time. This output is used to
 build graphs, similar to what is seen in the Grafana Explore graph view.
 If you are querying metrics and just want the most recent data point
 (like what is seen in the Grafana Explore table view), then you should use
-the "instant-query" command instead.`)
+the "instant-query" command instead.
+
+Parallelization:
+
+You can download logs in parallel, there are a few flags which control this
+behaviour:
+
+	--parallel-duration
+	--parallel-max-workers
+	--part-prefix
+	--overwrite-completed-parts
+	--merge-parts
+	--keep-parts
+
+Refer to the help of these specific flags to understand what each of them do.
+
+Example:
+
+	logcli query
+	   --timezone=UTC
+	   --from="2021-01-19T10:00:00Z"
+	   --to="2021-01-19T20:00:00Z"
+	   --output=jsonl
+	   --parallel-duration="15m"
+	   --parallel-max-workers="10"
+	   --part-prefix="/tmp/my_query"
+	   --merge-parts
+	   'my-query'
+
+This will start 10 workers, and they will each start downloading 15 minute
+slices of the specified time range.
```

Comment:
They won't finish after the first part though, right? I mean if we need more than ten 15m slices they would continue downloading.

Reply:
I'm not really sure what you are suggesting here. It's a pretty simple worker pool implementation. A worker will pick up the next job when it's done.

Reply:
I probably phrased it weird. I meant that the doc should sind the misconception that only ten works process then jobs. But maybe that's just me.

Reply:
I reworked the docs here, so I explain the whole process now. Hopefully it's as clear as it can be.